### ISSUE TITLE
Licensing topic: 'npm config set key value' syntax reported not to work for npm version 9+

### ISSUE THREAD

**Issue by ddaribo** [IgniteUI/igniteui-docfx#3874]
A couple of customers have reported that the 
`npm config set @infragistics:registry https://packages.infragistics.com/npm/js-licensed/`

command is no longer working for them (using npm version 9+).
The command works with a `=` sing before the link though:

`npm config set @infragistics:registry=https://packages.infragistics.com/npm/js-licensed/`

The npm config documentation claims to support the `npm config set key value` syntax in version 9+ for backwards compatibility ([link](https://docs.npmjs.com/cli/v9/commands/npm-config?v=true#set)), however, as the customers report, this is not working for them.
Having this in mind, the command snippet should probably be updated?

**Comment by damyanpetev** [comment]
Right, since 15.1.x is still in active support that means Angular 15 which still supports Node.js 14 https://angular.io/guide/versions#actively-supported-versions, which in turn is the last version that bundled npm 6 https://nodejs.org/en/download/releases.
Unfortunately, npm 6 from what I see was the one using the older API our docs use, so we can't quite replace it just yet. However, we could and probably should add the alternative - something in the lines of "For npm versions 7+" use the updated set call. I'd add we should test that works reliably from 7+ and if not list 8/9+ only.
//2c
